core/api/src/services/mongoose/wallet-invoices.ts

```diff
@@ -135,4 +137,5 @@ const walletInvoiceFromRaw = (result: WalletInvoiceRecord): WalletInvoice => ({
   paid: result.paid as boolean,
   usdAmount: result.cents ? UsdPaymentAmount(BigInt(result.cents)) : undefined,
   createdAt: new Date(result.timestamp.getTime()),
+  paymentRequest: result.paymentRequest as EncodedPaymentRequest,
```

Comment:
this is a casting of a potentially null/undefined value to string. this is pretty dangerous.
I would do this

```suggestion
  paymentRequest: result.paymentRequest ?? "unknown" as EncodedPaymentRequest,
```

Reply:
actually it's not because `WalletInvoice` marks `paymnetRequest` as optional... I still it make sense to return a string here.

Reply:
It will eventually be non nullable after a migration. But for now I think it makes sense to keep it undefined so we dont accidentally think we have a value when we dont.